Add ElevenLabs icon

z-alsamarai opened the pull request:
![elevenlabs](https://github.com/user-attachments/assets/947cdd53-b2b9-4829-91ad-e92888f328d9)

**Issue:** closes NA

**Popularity metric:**
* Similarweb Global Rank: [3,665](https://www.similarweb.com/website/elevenlabs.io/)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description
  - Picked Hex value 000000, because ElevenLabs uses a classic black and white color scheme, and it's the main color of their logo.
  - Used boxy-svg to resize to the proper viewbox, and used svgomg for optimization.

uncenter commented:
Please see the linting errors, your SVG isn't properly formatted. Additionally, we should use https://elevenlabs.io/icon.svg as the source as it is already an SVG and we don't need to vectorize.

z-alsamarai commented:
@uncenter My apologies, I pushed my commit before I saw your comment.
The SVG should be properly formatted now, and I tried using the SVG from the [link](https://elevenlabs.io/icon.svg) provided, it seems to yield the same result after removing all extraneous elements such as the rect in their version and scaling the viewbox down to 24x24.

uncenter commented:
>  it seems to yield the same result after removing all extraneous elements such as the rect in their version and scaling the viewbox down to 24x24.

What was the source you used before then? If you traced the PNG I find it hard to believe the SVG version lines up exactly 😅

LitoMore reviewed (changes requested):
I can see some differences.

![](https://github.com/user-attachments/assets/07148d81-4711-4481-be52-08b2a57e3f8d)

Can you check this?

z-alsamarai commented:
@LitoMore You are absolutely right, now I see it! This is my first time working with icons, and SVGs this way. 

Here's what I changed:

- Downloaded https://elevenlabs.io/icon.svg
- Modified the following to satisfy the guidelines: 

1. Removed the rectangle
2. Merged the two paths into one
3. Remove the grouping so the svg is just a single path
4. Made the viewbox 24x24
5. Centered the new path in the viewbox so that the top and bottom sides touch the border.
6. Optimized the new icon with svgomg

For reference, here's the original Icon elements:
![image](https://github.com/user-attachments/assets/39f6d783-ded6-4be5-909a-d9c92508aacd)

I hope this solves what you're looking for! If not I'm happy to make the necessary changes to make it work.

uncenter commented on _data/simple-icons.json:
```diff
@@ -4842,6 +4842,11 @@
+        {
+            "title": "ElevenLabs",
+            "hex": "000000",
+            "source": "https://elevenlabs.io/icon.svg"
```
```suggestion
            "source": "https://elevenlabs.io/brand",
            "guidelines": "https://elevenlabs.io/brand"
```

uncenter commented on icons/elevenlabs.svg:
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>ElevenLabs</title><path d="M19.314 24H14.4V0h4.914v24ZM9.6 24H4.686V0H9.6v24Z"/></svg>
```
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>ElevenLabs</title><path d="M4.6035 0v24h4.9317V0zm9.8613 0v24h4.9317V0z"/></svg>
```

The icon they provide on the brand page is apparently slightly different then the one from the previous source.
